Makefile

```diff
@@ -163,6 +167,13 @@ $(PROMETHEUS): | $(BIN_DIR)
 	@echo "Downloading Prometheus"
 	curl -L "https://github.com/prometheus/prometheus/releases/download/v$(PROMETHEUS_VERSION)/prometheus-$(PROMETHEUS_VERSION).$$(go env GOOS)-$$(go env GOARCH).tar.gz" | tar --strip-components=1 -xzf - -C $(BIN_DIR)
 
+$(LOKI): | $(BIN_DIR)
+	@echo "Downloading Loki"
+	(cd $(BIN_DIR) && curl -O -L "https://github.com/grafana/loki/releases/download/v$(LOKI_VERSION)/loki-linux-$$(go env GOARCH).zip" && \
+	unzip loki-linux-$$(go env GOARCH).zip && \
+	mv loki-linux-$$(go env GOARCH) loki && \
+	rm loki-linux-$$(go env GOARCH).zip)
```

Comment:
While we set GOARCH we don't set GOOS, this is going to break for people who would like to build on a Mac, I think.